### ISSUE TITLE
[Request] scoop manifest for window ?  

### ISSUE THREAD

**Issue by omaru-ok** [solidiquis/erdtree#36]
i would really love if I can have 
[scoop](scoop.sh) manifest , so it would be easy to install on window 

THX

**Comment by solidiquis** [comment]
I'll look into this during the weekend :]

**Comment by omaru-ok** [comment]
@solidiquis  project is getting regular updates  , 
it should be good if `scoop` way would be ready 

THX

**Comment by solidiquis** [comment]
Yeah I apologize for being slow with this.. things have been quite chaotic for me since the silicon valley bank fiasco. I'll look into it soon!

**Comment by fawni** [comment]
i'm currently working on this. i will make a pr to add scoop to the readme as soon as it gets merged!

**Comment by solidiquis** [comment]
@fawni you are amaze <3
